service/kvmconfig/metric/metric.go

```diff
@@ -0,0 +1,24 @@
+package metric
```

Comment:
I had to create a new package to avoid cycles. Still `metric.VersionBundleVersionGauge` reads nice and package makes sense (at least for me :) ).

Reply:
I think this is good for now, but if we have similar gauges more in projects, would it make sense to extract these to a library where you could just issue them with one function call?

Reply:
I didn't think about that. Right now it's a single metric. I'd say we should start to think about it when we have ~10 common metrics. One of the problem is that namespace change, but probably not very challenging.